eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig

```diff
@@ -9,6 +9,16 @@
 <ul class="ez-fielddefinition-settings ez-fielddefinition-{{ fielddefinition.fieldTypeIdentifier }}-settings">
     {% set defaultValue = fielddefinition.defaultValue.text %}
     {{ block( 'settings_defaultvalue' ) }}
+    <li class="ez-fielddefinition-setting min-length">
+        <div class="ez-fielddefinition-setting-name">Min string length:</div>
+        <div class="ez-fielddefinition-setting-value">
+        {% if fielddefinition.validatorConfiguration.StringLengthValidator.minStringLength %}
+            {{ fielddefinition.validatorConfiguration.StringLengthValidator.minStringLength }} characters
```

Comment:
there's no translation on field definition settings in 6.7 (1.7), this was added in 6.8 (1.8) so I'll add them while forward porting this change in 6.8.